x-pack/test/functional/config.js

```diff
@@ -448,6 +448,12 @@ export default async function ({ readConfigFile }) {
         global_upgrade_assistant_role: {
           elasticsearch: {
             cluster: ['manage'],
+            indices: [
```

Comment:
I found this was necessary in order to create the indices to trigger deprecations